### DISCUSSION TITLE
Hello community, I would like to ask how to modify the volume of distribution of the human body？

### DISCUSSION THREAD

**Discussion by Scorrt542** [Open-Systems-Pharmacology/Forum#1832] [Questions & Problems]
![image](https://github.com/user-attachments/assets/8fcd5bee-6f1b-40ff-9e5a-424acb75fa74)
Just like the one in this article Vd(L),Physiologically‑based pharmacokinetic/ pharmacodynamic modeling of meropenem in critically ill 
patients.(DOI:https://doi.org/10.1038/s41598-024-64223-0)

**Comment by michielve** [comment]
Hi,
I am just a user of PK-Sim, but this is what I understand:
The volume of distribution is being calculated based on the selected method for calculating the partition coefficients (e.g. PK-Sim Standard, Poulin and Theil, etc.), see the following page for explanation of each [method](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow) and significantly by the lipophilicity.

In order to optimize the Vd (i.e. select the best fitting method) you can go into your parameter identification run, select the configuration tab, from the Option menu select 'Calculation methods variation', select checkbox for 'All the same', select checkbox for 'Partition coefficients', and select the five methods. As a parameter, you can add the Lipophilicity to optimize.

After clicking on run, the model will try your parameters based on every calculation method and the lowest error in the results will be the best method with the optimized Vd.

**Reply by Scorrt542** [reply]
Hello, thank you for your answer, I'll try
